### PR TITLE
Created README.md

### PR DESCRIPTION
**Pull request by aRealCyborg**
Made a read me for the Lost gameplay template.
Explaining and showing the features and goal of the this gameplay template.

### CODE REVIEW

File: README.md
Comment (jdrueckert) [inline README.md]:
```suggestion
Download the Lost module and it's dependencies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.
```

File: README.md
Comment (jdrueckert) [inline README.md:6]: Consider adding a note on that the structures (the small house and the puzzle structures to find when exploring) are generated during the game. So sometimes visiting a location twice can be useful. Also, - in case it's not in the book of secrets texts (I'm not sure about that) - mentioning that different times of day might help to find different puzzles might make sense.

You can use collapsible text (like in https://gist.github.com/joyrexus/16041f2426450e73f5df9391f7f7ae5f) to hide potential spoilers.

File: README.md
Comment (jdrueckert) [inline README.md]: Not sure what you mean here?

**Review by jdrueckert — DISMISSED** [review]
Nice work in general and I like the use of screenshots without spoiling too much of the game content :+1:

File: README.md
Comment (aRealCyborg) [reply README.md]: ahh I knew I would misspell something!

File: README.md
Comment (aRealCyborg) [reply README.md]: I was just guessing, I haven't tested Lost enough to know about the set pieces that spawn.

Comment (aRealCyborg) [reply README.md]: Replacing section with better details

File: README.md
Comment (aRealCyborg) [reply README.md:6]: ok will take note.

File: README.md
Comment (Cervator) [inline README.md]: "It's" vs "Its"

```suggestion
Download the Lost module and its dependencies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.
```

File: README.md
Comment (Cervator) [inline README.md]: Another typo. It really helps to use an editor that automatically checks for spelling and even grammar / issues like the its vs it's bit - even if you're editing in a browser you can usually get addons for that :-)

```suggestion
As you explore you might find buildings that contain treasures, secrets and puzzles.
```

**Review by Cervator — DISMISSED** [review]
Readme overall seems fine, couple more entirely trivial typos but I can just adjust those while merging, not really worth another review cycle 👍

**Review by Cervator — APPROVED** [review]
Re-approving after applying suggestions so I can squash merge via web - could always do more but it is also a balancing act with a relatively small amount of stuff to cover (and some of it spoilery).

**Pull request merged**
